extensions/apple-mediakeys/chrome/content/prefsOverlay.xul

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+
+<!DOCTYPE overlay SYSTEM "chrome://apple-mediakeys/locale/options.dtd">
+
+<overlay id="radio-prefs-overlay"
+         xmlns="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul">
+
+  <prefWindow id="BrowserPreferences">
+    <prefpane id="mediakeysPrefPane"
+              label="&apple-mediakeys-prefs-overlay.label;"
+              src="chrome://apple-mediakeys/content/prefsPane.xul" />
+ </prefWindow>
```

Comment:
Can't you do this by changing the pref type? Or is this because the extension is hidden by default?